openpype/hosts/fusion/hooks/pre_pyside_install.py

```diff
@@ -0,0 +1,152 @@
+import os
+import subprocess
+import platform
+from openpype.lib.applications import PreLaunchHook, LaunchTypes
+
+
+class InstallPySideToFusion(PreLaunchHook):
+    """Automatically installs Qt binding to fusion's python packages.
+
+    Check if fusion has installed PySide2 and will try to install if not.
+
+    For pipeline implementation is required to have Qt binding installed in
+    fusion's python packages.
+    """
+
+    app_groups = {"fusion"}
+    order = 2
+    launch_types = {LaunchTypes.local}
+
+    def execute(self):
+        # Prelaunch hook is not crucial
+        try:
+            self.inner_execute()
+        except Exception:
+            self.log.warning(
+                "Processing of {} crashed.".format(self.__class__.__name__),
+                exc_info=True
+            )
+            raise
```

Comment:
Why is this try except block needed?

Reply:
That raise shouldn't be there, forgotten from debugging.

Reply:
Likely it's there to _never_ block the actual launch of Fusion itself, even if the PySide install failed for whatever reason in this code.